src/pages/index.js

```diff
@@ -67,6 +67,19 @@ const IndexPage = ({data}) => {
 
         <StatesList stateSlugs={stateSlugs}/>
       </div>
+
+      <hr className="mt-7" />
+
+      <section className="text-center mb-8">
+        <br className="d-none d-lg-block" />
+        <div className="h1 font-weight-bold">Ready to do your part now?</div>
```

Comment:
Can you change this to an `h2`? It's a small bit of text underneath, but this is indeed a heading

Reply:
done